### ISSUE TITLE
Typo in BIXPIX?

### ISSUE THREAD

**Issue by kadrlica** [pmelchior/des-exp-checker#88]
@pmelchior should this line be `BITPIX` instead of `BIXPIX`?
https://github.com/pmelchior/des-exp-checker/blob/master/assets/fits.js#L237

**Comment by pmelchior** [comment]
Yeah, it should. The code isn't mine, though. It came from Amit Kapadia's FITS implementation in JS.
